deeplake/enterprise/dataloader.py

```diff
@@ -16,7 +16,8 @@
 from deeplake.util.dataset import map_tensor_keys
 from functools import partial
 import importlib
-from torch.utils.data import DataLoader
+from torch.utils.data.dataloader import DataLoader, _InfiniteConstantSampler
```

Comment:
does this add a hard dependency on pytorch?

Reply:
this dependency is required only in case of using enterprise dataloader